tools/run_tests/stress_test/README.md

```diff
@@ -0,0 +1,72 @@
+﻿Running Stress tests on Google Container Engine
+=======================================
+
+### **Glossary**:
+* GCP: Google Cloud Platform
+* GCE: Google Compute Engine
+* GKE: Google Container Engine
+* GCP console: https://console.cloud.google.com
+
+### **Setup Instructions**
+#### *On GCP:*
+1. Create a GCP account (if you haven’t already) at https://cloud.google.com 
+2. Enable billing on Google cloud platform. Instructions [here](https://cloud.google.com/container-engine/docs/before-you-begin)  (see the '*Enable billing*' section).
+3. Create a Project from the [GCP console](https://console.cloud.google.com) 

```

Comment:
Add some detailed steps besides a link for first-time GCP users to reduce the time they need to spend on setup. For example it's not obvious for me on this page how to create a project.